server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -123,7 +123,6 @@ private ConfigProperties() {
      * will be filtered, meaning they will not enter HORNETQ.
      */
     public static final String AUDIT_FILTER_DEFAULT_POLICY = "candlepin.audit.filter.policy";
-    public static final String AUDIT_LOG_VERBOSE = "candlepin.audit.log_verbose";
```

Comment:
Can you verify that Candlepin will still start successfully with a configuration file that still has a `candlepin.audit.log_verbose` entry?  I just want to make sure that old configuration files don't cause a start-up error because they don't parse.

Reply:
Verified that there's no startup-error when deploying candlepin with ```candlepin.audit.log_verbose=true``` in /etc/candlepin/candlepin.conf

Reply:
💎 💰 👍